### PR TITLE
gitlab-ci: fixup evaluator results file with advisor results

### PR DESCRIPTION
**Pull request by unclejay80**
Evaluator should use the advisor results although the scanner is deactivated

Signed-off-by: Jürgen Wischer <33184634+unclejay80@users.noreply.github.com>

Please ensure that your pull request adheres to our [contribution guidelines](https://github.com/oss-review-toolkit/.github/blob/main/CONTRIBUTING.md). Thank you!

### CODE REVIEW

File: scripts/ort-evaluator.sh [hunk 1]
```diff
@@ -4,13 +4,16 @@
...
+    if [[ "$ORT_DISABLE_ADVISOR" = "false" ]]; then
+        ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_ADVISOR_FILE
+    else
+        ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_ANALYZER_FILE
+    fi
 else
     ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_SCANNER_FILE
 fi
 
+
```
Comment (sschuberth) [inline scripts/ort-evaluator.sh]: Nit: Unrelated change.

File: scripts/ort-evaluator.sh [hunk 2]
```diff
@@ -4,13 +4,16 @@
 # against customizable policy rules.
 
 if [[ "$ORT_DISABLE_SCANNER" = "true" ]]; then
-    ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_ANALYZER_FILE
-elif [[ "$ORT_DISABLE_ADVISOR" = "false" ]]; then
-    ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_ADVISOR_FILE
+    if [[ "$ORT_DISABLE_ADVISOR" = "false" ]]; then
```
Comment (sschuberth) [inline scripts/ort-evaluator.sh]: Nit: Let's stick to the same logic as one line above and compare with `= "true"`, swapping the code branches below.

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: scripts/ort-evaluator.sh (hunk 1 above)
Comment (sschuberth) [reply scripts/ort-evaluator.sh]: This unrelated line change is still present in the diff.

File: scripts/ort-evaluator.sh (hunk 2 above)
Comment (sschuberth) [reply scripts/ort-evaluator.sh]: I guess something went wrong while addressing this: The change per se is fine with me and should stay. Just the order of the introduced branches in the nested condition should be swapped, to match the logic of the surrounding condition.

Comment (unclejay80) [reply scripts/ort-evaluator.sh]: next try :) I hope this is fine now

File: scripts/ort-evaluator.sh (hunk 1 above)
Comment (unclejay80) [reply scripts/ort-evaluator.sh]: should be fine now

File: scripts/ort-evaluator.sh (hunk 2 above)
Comment (sschuberth) [reply scripts/ort-evaluator.sh]: Just some commit message tweaking 😉 Please use:

```
ort-evaluator: Use the correct results file

Use the advisor results file if the scanner is disabled but the advisor is
enabled.
```

(Plus your `Signed-of-by`, of course.)

Comment (unclejay80) [reply scripts/ort-evaluator.sh]: done

**Review by sschuberth — APPROVED** [review]
